### ISSUE TITLE
Allow going directly into select mode and selecting a given feature

### ISSUE THREAD

**Issue by gdnwr** [JamesLMilner/terra-draw#175]
As the title describes, I am currently simulating feature clicks through the onClick of SelectMode.
Use case: In a table, when clicking the edit button for a specific data entry, the corresponding data will be highlighted on the map, initiating the editing mode.
The current implementation is as follows:

``` TypeScript
   edit(feature: GeoJSON.Feature, callback?: Function) {
    this.terraDraw.start();
    this.terraDraw.setMode('select');

    const { type } = feature.geometry;
    let mode;
    if (type === 'Point') {
      mode = 'point';
    } else if (type === 'LineString') {
      //
    } else if (type === 'Polygon') {
      //
    }

    feature.properties = {
      ...(feature.properties || {}),
      mode,
    };
    this.terraDraw.addFeatures([feature as any]);

    let containerXY;
    if (type === 'Point') {
      containerXY = this.map.project(feature.geometry.coordinates);
      this.selectMode.onClick({
        lng: feature.geometry.coordinates[0],
        lat: feature.geometry.coordinates[1],
        containerX: containerXY.x,
        containerY: containerXY.y,
        button: 'left',
        heldKeys: [],
      });
    } else if (type === 'LineString') {
      //
    } else if (type === 'Polygon') {
      //
    }
  }
```

**Comment by JamesLMilner** [comment]
Hey @gdnwr, thanks for sharing your use case and existing work around! I think this is a totally valid scenario and I'd like to support being able to select via the API. I will have a think about the best way to allow for this to happen and update you with any progress.

**Comment by gdnwr** [comment]
Looking forward to this feature.

**Comment by JamesLMilner** [comment]
This is now available in release `0.0.1-alpha.57` 🌮 

You can just do `draw.selectFeature("9489d9ca-54f9-4a81-9ab4-3d2503cf9ea1")` with the id of the feature you want to select and do `draw.deselectFeature("9489d9ca-54f9-4a81-9ab4-3d2503cf9ea1")`.
